filetests/isa/x86/stack-load-store8.clif

```diff
@@ -0,0 +1,19 @@
+test compile
```

Comment:
Should this be `test legalizer`?

Reply:
I prefer `test compile` as it ensures that that all instructions after legalize do actually have encodings. For example https://github.com/bytecodealliance/cranelift/pull/1222 had a `test legalize` test which didn't actually compile.

Reply:
Sounds good to me.